packager/media/codecs/ac4_parser_unittest.cc

```diff
@@ -0,0 +1,87 @@
+// Copyright 2018 Google LLC. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+#include <packager/media/codecs/ac4_parser.h>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <packager/media/test/test_data_util.h>
+
+using ::testing::ElementsAre;
+
+namespace shaka {
+namespace media {
+
+    const uint8_t valid_ac4_data[] = {
```

Comment:
I'm not seeing the linter run yet because of a syntax error in Python, but I think this should not be indented.